src/java.base/share/classes/com/sun/crypto/provider/GaloisCounterMode.java

```diff
@@ -106,7 +106,7 @@ abstract class GaloisCounterMode extends CipherSpi {
     // Default value is 128bits, this is in bytes.
     int tagLenBytes = DEFAULT_TAG_LEN;
     // Key size if the value is passed, in bytes.
-    int keySize;
+    private final int keySize;
```

Comment:
Why not also mark the `blockCipher` as private final?

Reply:
Good catch!